test/test_python_solver_perf.py

```diff
@@ -37,13 +37,13 @@ class TestPythonSolverPerf(TestCase):
     def setUp(self) -> None:
         self.solvers = {
             NumPySSASolver: [
-                Tyson2StateOscillator()
+                build_tyson_2_state_oscillator()
             ],
             ODESolver: [
-                Oregonator()
+                build_oregonator()
             ],
             TauLeapingSolver: [
-                VilarOscillator()
+                build_vilar_oscillator()
             ],
         }
 
```

Comment:
`./test_python_solver_perf.py::TestPythonSolverPerf::test_python_solver_perf Failed with Error: A model is required to run the simulation.
  File "/usr/lib/python3.8/unittest/case.py", line 60, in testPartExecutor
    yield
  File "/usr/lib/python3.8/unittest/case.py", line 676, in run
    self._callTestMethod(testMethod)
  File "/usr/lib/python3.8/unittest/case.py", line 633, in _callTestMethod
    method()
  File "/home/mdip/Projects/GillesPy2/test/test_python_solver_perf.py", line 57, in test_python_solver_perf
    perf_data = python_profiler.run_profiler(model, solver)
  File "/home/mdip/Projects/GillesPy2/test/profiling/python_profiler.py", line 43, in run_profiler
    solver.run(number_of_trajectories=trajectories, t=timesteps, timeout=0)
  File "/home/mdip/Projects/GillesPy2/gillespy2/solvers/numpy/ssa_solver.py", line 90, in run
    self = NumPySSASolver(model=model)
  File "/home/mdip/Projects/GillesPy2/gillespy2/solvers/numpy/ssa_solver.py", line 38, in __init__
    raise SimulationError("A model is required to run the simulation.")
gillespy2.core.gillespyError.SimulationError: A model is required to run the simulation.`

Reply:
```
./test_python_solver_perf.py::TestPythonSolverPerf::test_python_solver_perf Failed with Error: A model is required to run the simulation.
  File "/usr/lib/python3.8/unittest/case.py", line 60, in testPartExecutor
    yield
  File "/usr/lib/python3.8/unittest/case.py", line 676, in run
    self._callTestMethod(testMethod)
  File "/usr/lib/python3.8/unittest/case.py", line 633, in _callTestMethod
    method()
  File "/home/mdip/Projects/GillesPy2/test/test_python_solver_perf.py", line 57, in test_python_solver_perf
    perf_data = python_profiler.run_profiler(model, solver)
  File "/home/mdip/Projects/GillesPy2/test/profiling/python_profiler.py", line 43, in run_profiler
    solver.run(number_of_trajectories=trajectories, t=timesteps, timeout=0)
  File "/home/mdip/Projects/GillesPy2/gillespy2/solvers/numpy/ssa_solver.py", line 90, in run
    self = NumPySSASolver(model=model)
  File "/home/mdip/Projects/GillesPy2/gillespy2/solvers/numpy/ssa_solver.py", line 38, in __init__
    raise SimulationError("A model is required to run the simulation.")
gillespy2.core.gillespyError.SimulationError: A model is required to run the simulation.
```